lib/pages/new_private_chat/widget/expansion_contact_list_tile.dart

```diff
@@ -21,15 +21,19 @@ class ExpansionContactListTile extends StatelessWidget {
   @override
   Widget build(BuildContext context) {
     return Padding(
-      padding: const EdgeInsets.only(left:8.0, top: 8.0, bottom: 12.0),
+      padding: const EdgeInsets.only(left: 8.0, top: 8.0, bottom: 12.0),
       child: Row(
         crossAxisAlignment: CrossAxisAlignment.start,
         children: [
-          RoundAvatar(
-            text: contact.displayName ?? "@",
-            size: 48,
+          FutureBuilder<Uri?>(
+            future: getAvatarUrl(context),
+            builder: (context, snapshot) {
+              return Avatar(mxContent: snapshot.data, name: contact.displayName);
+            },
```

Comment:
Be careful with the `FutureBuilder`, it maybe makes the blink blink effect

Reply:
Mxcimage puts a default avatar before data is loaded. It's already handled
I could remove the fade animation if you want tho

![blinkavatar](https://github.com/linagora/twake-on-matrix/assets/48354990/549e7269-1f4d-453a-bd82-f1ab6c004460)